change/@microsoft-fast-foundation-3d94788e-5d6d-4096-b102-e54c4d5cd2c3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I believe these should be `"type": "none"` because these changes don't impact the published packages.